### ISSUE TITLE
E-Mails löschen in Müll

### ISSUE THREAD

**Issue by frag-seb** [yakamara/redaxo_prozer#261]
Wenn E-Mails nicht beim Herunterladen gelöscht werden, kann man sie nicht richtig über den Mülleimer löschen. Sie tauchen dann einfach wieder in der Inbox auf.

**Comment by dergel** [comment]
stimmt.. habe noch keinen Lösungsansatz.. Irgendwie müsste man dann sich diese Mails merken.. Prozer ist da irgendwie nicht für ausgelegt.

**Comment by frag-seb** [comment]
Bei mir ist die Lösung nun im User die Einstellung "E-Mails nach dem Herunterladen löschen".

Nicht ideal aber so kann man es nutzen...
